packages/bitcore-wallet-service/src/lib/server.ts

```diff
@@ -4538,11 +4538,23 @@ export class WalletService {
   private moonpayGetKeys(req) {
     if (!config.moonpay) throw new Error('Moonpay missing credentials');
 
-    let env = 'sandbox';
-    if (req.body.env && req.body.env == 'production') {
-      env = 'production';
+    let env: 'sandbox' | 'production' | 'sandboxWeb' | 'productionWeb';
+    if (req.body.context && req.body.context === 'web') {
+      if (req.body.env && req.body.env === 'production') {
+        env = 'productionWeb';
+      } else {
+        env = 'sandboxWeb';
+      }
+    } else {
+      if (req.body.env && req.body.env === 'production') {
+        env = 'production';
+      } else {
+        env = 'sandbox';
+      }
```

Comment:
This can be simplified into:

```
    // production | sandbox | productionWeb | sandboxWeb
    let env = req.body.env === 'production' ? 'production' : 'sandbox';
    if (req.body.context === 'web') {
      env += 'Web';
    }
```